cwms-data-api/src/main/java/cwms/cda/data/dao/ClobDao.java

```diff
@@ -128,17 +101,17 @@ public Clobs getClobs(String cursor, int pageSize, String officeLike,
         }
 
         SelectLimitPercentStep<Record4<String, String, String, String>> query = dsl.select(
-                        v_office.OFFICE_ID,
-                        v_clob.ID,
-                        v_clob.DESCRIPTION,
-                        includeValues ? v_clob.VALUE : DSL.inline("").as(v_clob.VALUE)
-                )
-                .from(v_clob)
-                //.innerJoin(forLimit).on(forLimit.field(v_clob.ID).eq(v_clob.ID))
-                .join(v_office).on(v_clob.OFFICE_CODE.eq(v_office.OFFICE_CODE))
-                .where(JooqDao.caseInsensitiveLikeRegex(v_clob.ID,idRegex))
-                .and(DSL.upper(v_clob.ID).greaterThan(clobCursor))
-                .orderBy(v_clob.ID).limit(pageSize);
+                v_office.OFFICE_ID,
+                v_clob.ID,
+                v_clob.DESCRIPTION,
+                includeValues ? v_clob.VALUE : DSL.inline("").as(v_clob.VALUE)
+            )
+            .from(v_clob)
+            .join(v_office).on(v_clob.OFFICE_CODE.eq(v_office.OFFICE_CODE))
+            .where(whereClause)
+            .and(DSL.upper(v_clob.ID).greaterThan(clobCursor))
```

Comment:
This still isn't quite right. At least in my opinion.  If you are ordering by office and clobId then I think the paging has to do the sameOffice/nextOffice thing.  See buildPagingCondition in TimeSeriesDaoImpl for what I'm talking about

Reply:
I'm fairly certain Ryan is correct. *especially* if you have more than one office possibly coming through, but even just limiting to a single you want that sorting to be consistent.

Reply:
yup, thanks for catching this. Updated to match how you handled it in the projects endpoint

Reply:
Is there another commit? I don't see it in these changes